osclib/pkglistgen_comments.py

```diff
@@ -145,7 +184,13 @@ def is_approved(self, comment, comments):
                     return c['who']
         return None
 
-    def parse_title(self, line):
+    def parse_title(self, line: str) -> Optional[Dict[str, Union[str, List[str]]]]:
```

Comment:
If you want this typed, you should make the commands a class. Shouldn't be a lot of lines